permissions/plugin-cloudify.yml

```
@@ -0,0 +1,14 @@
+---
+name: "cloudify"
+github: "jenkinsci/cloudify-plugin"
+paths:
+- "io/jenkins/plugins/cloudify-plugin"
```

```suggestion
- "io/jenkins/plugins/cloudify"
```

It should be groupId / artifactId

---

Done